### DISCUSSION TITLE
Input multiple parameters in GDM3_CONFIG

### DISCUSSION THREAD

**Discussion by hc2021hc** [ltsp/ltsp#492] [General]
May I know if it is possible to input multiple parameters in GDM3_CONFIG, such as the following to LTSP clients:

TimedLoginEnable=true
TimedLogin=XXX
TimedLoginDelay=10

I tried \n and ; to separate the parameters but in vain.

**Comment by alkisg** [comment]
ltsp.conf is essentially a shell script, so the syntax for multiline variables is:

```shell
VAR="line1
line2
line3"
```

**Comment by hc2021hc** [comment]
Yes, it works.  Thanks so much for your help.
